opencensus/proto/agent/common/v1/common.proto

```diff
@@ -0,0 +1,95 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// NOTE: This proto is experimental and is subject to change at this point.
+// Please do not use it at the moment.
+
+package opencensus.proto.agent.common.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option java_multiple_files = true;
+option java_package = "io.opencensus.proto.agent.common.v1";
+option java_outer_classname = "CommonProto";
+
+option go_package = "github.com/census-instrumentation/opencensus-proto/gen-go/agentproto/commonproto/v1";
+
+// Identifier metadata of the Node that connects to OpenCensus Agent.
+// In the future we plan to extend the identifier proto definition to support
+// additional information (e.g cloud id, monitored resource, etc.)
+message Node {
+  // Identifier that uniquely identifies a process within a VM/container.
+  ProcessIdentifier identifier = 1;
+
+  // Information on the OpenCensus Library who initiates the stream.
+  LibraryInfo library_info = 2;
+
+  // Additional attributes.
+  map<string, string> attributes = 3;
+
+  // Additional informantion on service.
+  ServiceInfo service_info = 4;
+
+  // TODO(songya): Add more identifiers in the future as needed, like cloud identifiers.
+}
+
+// Identifier that uniquely identifies a process within a VM/container.
+message ProcessIdentifier {
+
+  // The host name. Usually refers to the machine/container name.
+  // For example: os.Hostname() in Go, socket.gethostname() in Python.
+  string host_name = 1;
+
+  // Process id.
+  uint32 pid = 2;
+
+  // Start time of this ProcessIdentifier. Represented in epoch time.
+  google.protobuf.Timestamp start_timestamp = 3;
+
+  // TODO(songya): Add more fields in the future as needed.
+}
+
+// Information on OpenCensus Library.
+message LibraryInfo {
+
+  enum Language {
+    UNKNOWN = 0;
```

Comment:
To be consistent with other enums in this repo maybe consider LANGUAGE_UNSPECIFIED

Reply:
SG, done.